### PR TITLE
Add platform support matrix

### PR DESCRIPTION
**Pull request by dthiesse**
Adding Platform Support Matrix to site. First step is just an image. Will create html table in next phase.

### CODE REVIEW

File: src/data/nav-items.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 - title: Playbook Home
   pages:
     - path: /playbook/
+- title: Cloud Platform Support Matrix
```
Comment (gangchen03) [inline src/data/nav-items.yaml]: @dthiesse can we change the title to the validation matrix. And in the page, we need to add statement: The following platform have been validated by IBM garage team.

**Comment by dthiesse** [comment]
My second commit addresses your requested changes @gangchen03 .

**Comment by dthiesse** [comment]
Third commit is just to remove an extra index.mdx file I had left behind while making the platform matrix changes.

**Comment by dthiesse** [comment]
PR Closed because of decision to expose the platform matrix internally instead of externally.

**Pull request closed without merging**